### ISSUE TITLE
you're missing code comments

### ISSUE THREAD

**Issue by Razpudding** [TuriGuilano/twitter-realtime#6]
How is somebody who hasn't worked with the twitter package supposed to know [what this does](https://github.com/TuriGuilano/twitter-realtime/blob/master/app.js#L31)? Document your API and code in general.

**Comment by TuriGuilano** [comment]
I agree.. will add comments and a clear explanation in my readme how I use all the packages etc.
